src/components/rating/subcomponents/Star.jsx

```diff
@@ -3,19 +3,21 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import Icon, {TYPE, ICON_COLOR} from '../../icons/Icon';
 
-const Star = ({size, checked, onClick}) => {
+const Star = ({size, checked, onClick, index}) => {
   const starClass = classnames('sg-rate-box__star', {
     'sg-rate-box__star--checked': checked
   });
+  const onClickStar = () => onClick(index);
```

Comment:
again, can we create these functions in parent?

Reply:
+1